src/lib.rs

```diff
@@ -683,10 +678,23 @@ impl StacksDevnetApiK8sManager {
                         user_id,
                         StacksDevnetPod::StacksBlockchainApi,
                     ),
-                    self.get_stacks_v2_info(&namespace),
```

Comment:
I just lack knowledge here; can you explain why removing it?

Reply:
The function is still being called below. The reason for the change in approach is to handle errors being thrown in `get_stacks_v2_info` and use default values for the chain tip info. This was causing the `GET /api/v1/networks/<name>` request to respond with 500 when the devnet chain was still booting up and unavailable, when we want to fetch the status information during loading.